infoblox_client/__init__.py

```diff
@@ -1,3 +1,3 @@
 __author__ = 'John Belamaric'
 __email__ = 'jbelamaric@infoblox.com'
-__version__ = '0.4.21'
+__version__ = '0.4.22'
```

Comment:
you have missed it here

Reply:
it should be 0.4.21

Reply:
Sorry, I'll remove last two commits (back to a24ad1a)